### ISSUE TITLE
Can't generate tsdf2mesh executable when I switch pcl version to pcl 1.10

### ISSUE THREAD

**Issue by Junkai-Liu** [sdmiller/cpu_tsdf#50]
When I compile CMakeLists which depend on pcl1.10,the output error as follow: 

Could NOT find PCL_COMMON (missing: PCL_COMMON_LIBRARY) 
-- Could NOT find PCL_KDTREE (missing: PCL_KDTREE_LIBRARY) 
-- Could NOT find PCL_OCTREE (missing: PCL_OCTREE_LIBRARY) 
-- Could NOT find PCL_SEARCH (missing: PCL_SEARCH_LIBRARY) 
-- Could NOT find PCL_SAMPLE_CONSENSUS (missing: PCL_SAMPLE_CONSENSUS_LIBRARY) 
-- Could NOT find PCL_FILTERS (missing: PCL_FILTERS_LIBRARY) 
-- Found PCL_2D: /usr/local/include/pcl-1.10  
-- Found PCL_GEOMETRY: /usr/local/include/pcl-1.10  
-- Could NOT find PCL_IO (missing: PCL_IO_LIBRARY) 
-- Could NOT find PCL_FEATURES (missing: PCL_FEATURES_LIBRARY) 
-- Could NOT find PCL_ML (missing: PCL_ML_LIBRARY) 
-- Could NOT find PCL_SEGMENTATION (missing: PCL_SEGMENTATION_LIBRARY) 
-- Could NOT find PCL_VISUALIZATION (missing: PCL_VISUALIZATION_LIBRARY) 
-- Could NOT find PCL_SURFACE (missing: PCL_SURFACE_LIBRARY) 
-- Could NOT find PCL_REGISTRATION (missing: PCL_REGISTRATION_LIBRARY) 
-- Could NOT find PCL_KEYPOINTS (missing: PCL_KEYPOINTS_LIBRARY) 
-- Could NOT find PCL_TRACKING (missing: PCL_TRACKING_LIBRARY) 
-- Could NOT find PCL_RECOGNITION (missing: PCL_RECOGNITION_LIBRARY) 
-- Could NOT find PCL_STEREO (missing: PCL_STEREO_LIBRARY) 
-- Could NOT find PCL_OUTOFCORE (missing: PCL_OUTOFCORE_LIBRARY) 
-- Could NOT find PCL_PEOPLE (missing: PCL_PEOPLE_LIBRARY) 

After build process, the project can generate shared library , however, can't generate the tsdf2mesh executable file. 
By the way, If I compile with pcl 1.8, pcl/surface/include/pcl/surface/marching_cubes.h can't definite protected data:  size_voxel_   upper_boundary_   lower_boundary_
It will occur the bug : cant find the deifnition of the above protected data.
So, the project depend on which version of pcl?

**Comment by sdmiller** [comment]
Hi @Junkai-Liu,

As before, I apologize for never having addressed this in realtime. I've most recently tested this on PCL 1.10, 1.11, and 1.12 without issue, though it was originally developed against PCL 1.7. If you are still in need of this tool and having trouble, please open a new issue and I will do my best to address it in a more timely manner!
